src/main/java/weavers/siltarae/comment/domain/Comment.java

```diff
@@ -0,0 +1,39 @@
+package weavers.siltarae.comment.domain;
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import weavers.siltarae.global.BaseEntity;
+import weavers.siltarae.member.domain.Member;
+import weavers.siltarae.mistake.domain.Mistake;
+
+import static lombok.AccessLevel.PROTECTED;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = PROTECTED)
+public class Comment extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.DETACH, fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id")
+    private Member member;
+
+    @Column(nullable = false, length = 140)
+    private String content;
+
+    @ManyToOne(cascade = CascadeType.DETACH, fetch = FetchType.LAZY)
+    @JoinColumn(name = "mistake_id")
+    private Mistake mistake;
```

Comment:
Mistake는 피드에서 댓글 없이 미리보기가 가능할 것 같은데,
Comment가 영속성 컨텍스트에서 DETACH 될 때 Mistake까지 같이 DETACH될 필요가 있을까요?

Reply:
그렇겠네요 Mistake 가 삭제가 될때를 생각했었는데 제가 반대로 생각하고 있었네요..ㅎ 😅

```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "mistake_id")
    private Mistake mistake;
```